### PR TITLE
Add ruby 2.2 and 2.3 to travis build

### PR DESCRIPTION
**Pull request by artofhuman**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -5,4 +5,5 @@ gemspec
 group :test do
   gem "codeclimate-test-reporter", require: false
   gem "rspec", "~> 3.1"
+  gem "json", "< 2.0.0" if RUBY_VERSION < '2'
```
Comment (laserlemon) [inline Gemfile]: Is this necessary to green the suite? Was Ruby 1.9.3 failing prior to this?

File: .travis.yml [hunk 2]
```diff
@@ -18,5 +18,7 @@ rvm:
   - 1.9.3
   - "2.0"
   - "2.1"
+  - "2.2"
+  - "2.3"
```
Comment (laserlemon) [inline .travis.yml]: Is there any reason not to include Ruby 2.4 now as well?

File: Gemfile (hunk 1 above)
Comment (artofhuman) [inline Gemfile]: now it not neccessary https://github.com/flori/json/commit/3015920d6eccc2885f5a9ac2d5531800c4bebc24

File: .travis.yml (hunk 2 above)
Comment (artofhuman) [inline .travis.yml]: 2.4 added

Comment (laserlemon) [inline .travis.yml]: Ugh, it failed on 2.4 because RVM doesn't seem to want to fuzzy match. We may have to be more specific for 2.4.

**Comment by artofhuman** [comment]
🍏

**Comment by laserlemon** [comment]
Thank you! 👏

**Pull request merged**